sdc-plugin/tests/get_clocks/get_clocks.tcl

```diff
@@ -1,8 +1,7 @@
 yosys -import
-plugin -i sdc
-plugin -i design_introspection
-# Import the commands from the plugins to the tcl interpreter
-yosys -import
+if { [info procs read_sdc] == {} } { plugin -i sdc }
+if { [info procs get_nets] == {} } { plugin -i design_introspection }
+yosys -import  ;# ingest plugin commands
```

Comment:
This would be an example where we first load a bunch of plugins, then finally `yosys -import`.